addons/mail/models/mail_template.py

```diff
@@ -747,14 +752,20 @@ def send_mail_batch(self, res_ids, force_send=False, raise_exception=False, emai
                     'model_description': model_lang.display_name,
                     'record': record,
                     'record_name': False,
-                    'subtitles': False,
+                    'subtitles': subtitles,
+                    'subtitles_highlight_2nd': subtitles_highlight_2nd,
                     # user / environment
                     'company': company,
                     'email_add_signature': False,
                     'signature': '',
                     'website_url': '',
                     # tools
                     'is_html_empty': is_html_empty,
+                    # display
```

Comment:
Code of template now use ``_render_encapsulate``, in order to have everything localized at the same place. And it has more default context / variables, to match what is done when doing message_post. Globally it should be easy to update :)